styles/styles.css

```diff
@@ -0,0 +1,100 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  background: linear-gradient(90deg, #e3ffe7 0%, #d9e7ff 100%);
+}
+
+#main-doc {
+  display: grid;
+  grid-template-columns: 300px 1fr;
+  grid-template-rows: 1fr;
+}
+
+.box {
+  grid-column-start: 1;
+  display: block;
+  position: fixed;
+  height: 100vh;
+  width: 300px;
+  overflow: hidden;
+  border-right: 4px solid black;
+}
+
+#navbar {
+  display: flex;
+  flex-direction: column;
+}
+
+a {
+  color: black;
+  font-size: 1.4rem;
+  padding: 0.8rem 0 0.8rem 1rem;
+  text-decoration: none;
+  transition: color 0.3s ease-in-out;
+}
+
+a:hover {
+  color: rgb(1, 1, 150);
+  font-weight: bold;
+}
+
+.container {
+  grid-column-start: 2;
+}
+
+.main-section {
+  padding: 1.3rem 1rem;
+}
+
+.h1 {
+  padding: 1rem 0 0 1rem;
+  font-size: 1.6rem;
+  font-weight: 700;
+}
+
+header h1,
+h2 {
+  padding: 1rem 0;
+  font-size: 2rem;
+}
```

Comment:
It doesn't affect your project, but in your HTML you didn't use an h1 tag, I think you should remove that from CSS. :+1: